packages/slate-react/src/components/editable.tsx

```diff
@@ -912,13 +912,25 @@ export const Editable = (props: EditableProps) => {
 
                 if (Hotkeys.isMoveWordBackward(nativeEvent)) {
                   event.preventDefault()
-                  Transforms.move(editor, { unit: 'word', reverse: !isRTL })
+
+                  if (selection && Range.isCollapsed(selection)) {
+                    Transforms.move(editor, { unit: 'word', reverse: !isRTL })
+                  } else {
+                    Transforms.collapse(editor, { edge: 'start' })
+                  }
```

Comment:
I like this change, but a simple collapse isn't _quite_ how the range selection scenario works, at least on macOS. It collapses and then moves.

```suggestion
                  if (selection && !Range.isCollapsed(selection)) {
                    Transforms.collapse(editor, { edge: 'start' })
                  }
                  
                  Transforms.move(editor, { unit: 'word', reverse: !isRTL })
```

Some apps collapse to the focus rather than the start, which is how I prefer to do it, but there is no consistency even amongst Apple apps so that's more of a personal choice.

Reply:
> * When moving towards the anchor, the selection is first collapsed at the anchor.

Single word selections can be misleading; try selecting 3 words and see what happens. My read of your video is that it is collapsing to the focus and then moving a word, but it isn't totally clear with just a single word.

Selecting multiple words left-to-right, then pressing alt+left, puts the cursor in the middle of the range that was selected.